RNFBSDKExample/README.md

```diff
@@ -4,6 +4,8 @@ This example is extremely basic but hopefully still useful.
 
 You run it by cloning the main repo, and running `yarn` then `yarn example:android` or `yarn example:ios`
 
+Note: If you hae run it before, and are trying to update it `rm yarn.lock node_modules ios/Podfile.lock && yarn && cd ios && pod install` in order to make sure you have fresh code, or if you are developing run `yarn example:copy` from the parent direcotry. Why? The example references the module code via a `../` reference, and the and it may contain stale files unless you clear the lockfiles/modules and reinstall.
```

Comment:
Just a typo here :)
What about having this command as an npm script also?

Reply:
Argh typos, will fix :-)

I did put `yarn example:copy` in the run scripts, I could put a `yarn example:clean` as well to package up the other ones, good idea. I'll add that and fix the typo and repush when I get a minute

Thanks for the review! With regard to documenting etc, I'm always trying to code for the "me" in the future (like, just a month or two for now) that will have forgotten all this and will need the information for some future bug (there's always a future bug...). Then the documentation and commits and comments become a valuable gift from Me-from-the-past :nerd_face: